x-pack/plugin/sql/qa/server/src/main/resources/docs/geo.csv-spec

```diff
@@ -15,7 +15,7 @@ selectAsWKT
 SELECT city, ST_AsWKT(location) location FROM "geo" WHERE city = 'Amsterdam';
 
      city:s    |     location:s
-Amsterdam      |POINT (4.850311987102032 52.347556999884546)
+Amsterdam      |POINT (4.850312 52.347557)
```

Comment:
Any idea where does the rounding come from?

Reply:
@costin  "fields" API is relying (atm) on extraction from _source, which is basically the input the user used when indexing. Before "fields" API we were relying mainly on extraction from _source but there were field types (geo_point and shape being two of them) that we were taking from docvalues (meaning the actual values that were indexed). And in docvalues, things are just a bit different when it comes to floating point numbers and geo stuff.

That specific document with Amsterdam city has been indexed as `"city": "Amsterdam", "location": "52.347557,4.850312"` (`location` is a `geo_point`) and the returned value is pretty much what has been actually sent as _source to ES.